v2/routes/instructorAPI.js

```diff
@@ -295,8 +315,8 @@ function instructorAPI(db, instructorAuth, isInstructor) {
 			let insertRes = await slides.insertOne({
 				course: course._id,
 				filename: req.files.file.name,
-				anonymity: req.body.anonymity,
-				drawable: true,
+				anonymity: course.anonymity,
```

Comment:
does this work? the query projection above does not contain anonymity or drawable